### PR TITLE
Enable signing on GH actions

### PR DESCRIPTION
**Pull request by himdel**
Testing how to get the signing service working in pulp-ci images on github actions... originally part of #2069 

docs https://galaxyng.netlify.app/config/collection_signing/

### CODE REVIEW

**Comment by himdel** [comment]
:tada: @brumik I think this is actually working now :) 

Or at least got as far as getting the feature flag true :) https://github.com/ansible/ansible-hub-ui/runs/7034195666?check_suite_focus=true#step:18:12

**Comment by himdel** [comment]
> Successfully added signing service ansible-default for key EBED170E8C9480E22A1D059B15250E9EC0A62577.
https://github.com/ansible/ansible-hub-ui/runs/7053975915?check_suite_focus=true#step:14:60

> `"can_create_signatures": true,`
https://github.com/ansible/ansible-hub-ui/runs/7053975915?check_suite_focus=true#step:19:15

The problem was that `collection_sign.sh` != `collection-sign.sh`, and the signing setup needs to run at in the running container, fails at container build time.

**Review by brumik — APPROVED** [review]
This works as expected, and allows us to test signing related features.

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -43,16 +43,27 @@ jobs:
       uses: actions/cache@v2
       with:
         path: pulp_galaxy_ng/image
-        key: ${{ runner.os }}-container-${{ env.GALAXY_NG_COMMIT }}-post788
+        key: ${{ runner.os }}-container-${{ env.GALAXY_NG_COMMIT }}-post2240-
```
[the remark this answers is not in the document]
Comment (brumik) [reply .github/workflows/cypress.yml:46]: It is confusing for me that the galaxy_ng_commit is mixed with `post{UI_pr_number}`

[the remark this answers is not in the document]
Comment (himdel) [reply .github/workflows/cypress.yml:46]: Ah, it's so we don't have to rebuild the backend container on every test run - so we're caching the container as long as the `key` is the same, and only rebuilding the container when a new commit gets merged in galaxy_ng.

But... if the key is just the galaxy_ng top commit hash, any change in HOW the container is being built gets ignored until we merge something in the backend.

So we're really using a compound key made of automatically retrieved galaxy_ng commit hash, and manually specified version of the build script here.

**Pull request merged**